README.md

```diff
@@ -17,6 +17,8 @@ This directory contains directly usable tools installable via:
 
 ```
 go get periph.io/x/cmd/...
```

Comment:
Oh dang, sorry about that, please remove.

Reply:
Will do